core/kits/documents/tabs/settings-layout.php

```diff
@@ -69,8 +70,18 @@ protected function register_tab_controls() {
 						'max' => 1500,
 						'step' => 10,
 					],
+					'%' => [
+						'min' => 50,
+						'max' => 100,
+						'step' => 1,
+					],
+					'vw' => [
+						'min' => 50,
+						'max' => 100,
+						'step' => 1,
+					],
```

Comment:
Remove the range values. Elementor will inherit this from `Control_Base_Units()` class.

Reply:
Done.